stylelint.config.js

```diff
@@ -0,0 +1,218 @@
+module.exports = {
+    extends: 'stylelint-config-sass-guidelines',
+    ignoreFiles: [
+        '**/*.js',
+    ],
+    rules: {
```

Comment:
Are these rules an industry standard like the airbnb JS linting rules, or is this a set of rules that you've found work well based on your own experience?